es.go

```diff
@@ -710,6 +710,18 @@ func (c *Client) GetIndices(index string) ([]Index, error) {
 	return indices, nil
 }
 
+// Get a subset of indices including hidden ones
+func (c *Client) GetHiddenIndices(index string) ([]Index, error) {
```

Comment:
Just for my understanding - what does hidden indices mean here? 🤔

Reply:
There are some indices created by ILM itself. These indices keep metadata about ILM. Users can also create their own hidden indices by setting correct flag during index generation. This will prevent these indices to appear while listing indices without proper flag.